### PR TITLE
Create January APICentral Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Created January Release Notes topic. Ready for content.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [x] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Comment by lbadenhop** [comment]
@cbordier Ready for your content

**Review by robertengaxway — APPROVED** [review]
The Central portion of the release notes looks accurate to me.  I approve

**Review by cbordier — APPROVED** [review]
Approved by all POs,
@lbadenhop, please proceed.

File: content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+**This is not yet compatible with the Axway CLI**.
+
+The Amplify Central CLI includes the following enhancements:  
+
+* Installation of Azure agents (use `amplify central install agents` using the new **Azure** option)
+* Installation of the alpha Mesh Governance Discovery Agent (use `amplify central install agents` using the new **Kubernetes** option)
+* Providers can use new resource shortnames to access resources (use `amplify central get` to view the list of all resource shortnames)
+* Providers can remove previously configured parameters using the interactive mode of the Central CLI (use `amplify central config unset <key>`)
+* Providers can request multiple resources at one time. The result will show the resource tables split according to their resouce kind (sample `amplify central get apis,apisr,apisi –s <scopename>`)
```
Comment (lbadenhop) [inline content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md:28]:
```suggestion
* Providers can request multiple resources at one time. The result displays the resource tables split according to resource type (sample `amplify central get apis,apisr,apisi –s <scopename>`)
```

File: content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+### Amplify Central WebUI
+
+The Amplify Central WebUI is used by both the API providers and consumers to manage and consume APIs.
+
+The Amplify Central WebUI includes the following enhancements:  
+
+* View the agents connected to an environment in environment detail page
```
Comment (lbadenhop) [inline content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md:36]:
```suggestion
* View the agents connected to an environment in the Environment Detail page
```

File: content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+* Improved Provider UX of API Service List and Details page:
+
+    * Providers can view the user who made the last modification from the list of API Service versions.
```
Comment (lbadenhop) [inline content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md:39]:
```suggestion
    * Providers can view the user who made the last modification from the list of API Service versions
```

File: content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    * Providers can search/sort on the Endpoints and Catalog Item list tables.
```
Comment (lbadenhop) [inline content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md:40]:
```suggestion
    * Providers can search/sort on the Endpoints and Catalog Item list tables
```

File: content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+    * Providers can add new API Services from the UI.
```
Comment (lbadenhop) [inline content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md:41]:
```suggestion
    * Providers can add new API Services from the UI
```

File: content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+### Axway Edge Gateway / AWS / Azure Agents
+
+To provide better visibility into your mutli-type gateway eco system, two sets of agents are provided. These agents collect data from the Gateway (API / traffic) and exposes it in Amplify Central, providing you with a global vision of your eco system from a single interface.
+
+The agents include the following enhancements:
+
+* Azure gateway support. Refer to [Connect Azure Gateway to Axway Amplify platform](https://docs.axway.com/bundle/axway-open-docs/page/docs/central/connect-azure-gateway/index.html)
```
Comment (lbadenhop) [inline content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md:49]:
```suggestion
* Azure Gateway support. Refer to [Connect Azure Gateway to Axway Amplify platform](/page/docs/central/connect-azure-gateway/index.html)
```

File: content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+* The Mesh  Governance open beta Traceability Agent has been updated to allow a set of header redaction rules to be passed to it on deployment. The rules apply globally to all transaction logging and provide options for path, query parameter and header filtering. Request and response headers may be separately managed to selectively redacted or entirely removed from the results visible in the API Observer.
+* Both of these Traceability agent options are applied using the mesh agent helm deployment step and are documented in Step 4 of the full mesh client and cluster setup instructions here: [https://github.com/Axway/Setup-Amplify-Mesh-Governance/wiki/Step-4.-Create-an-Amplify-Central-environment-and-connect-your-Kubernetes-cluster-to-it](https://github.com/Axway/Setup-Amplify-Mesh-Governance/wiki/Step-4.-Create-an-Amplify-Central-environment-and-connect-your-Kubernetes-cluster-to-it). Redeploying with helm to change either of the two options above will cause the mesh agent to restart.
```
Comment (lbadenhop) [inline content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md:61]:
```suggestion
  * Both of these Traceability agent options are applied using the mesh agent helm deployment step and are documented in Step 4 of the full mesh client and cluster setup instructions here: [https://github.com/Axway/Setup-Amplify-Mesh-Governance/wiki/Step-4.-Create-an-Amplify-Central-environment-and-connect-your-Kubernetes-cluster-to-it](https://github.com/Axway/Setup-Amplify-Mesh-Governance/wiki/Step-4.-Create-an-Amplify-Central-environment-and-connect-your-Kubernetes-cluster-to-it). Redeploying with helm to change either of the two options above will cause the mesh agent to restart.
```

File: content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+
+* The alpha Mesh Governance Discovery Agent can be installed with the CLI option: `amplify central install agents` using the new **Kubernetes** option. Note that the alpha Mesh Governance agent capabilities are separate from the existing beta agents, and are available for preview and feedback compared to the existing beta agents.
```
Comment (lbadenhop) [inline content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md:63]:
```suggestion
* The alpha Mesh Governance Discovery Agent can be installed with the CLI option: `amplify central install agents` using the new **Kubernetes** option. Note that the alpha Mesh Governance agent capabilities are separate from the existing beta agents, and are available for preview and feedback.
```

File: content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+* The alpha agent installation is documented here: [https://docs.axway.com/bundle/axway-open-docs/page/docs/central/mesh_management/deploy-your-agents-with-the-amplify-cli/index.html](https://docs.axway.com/bundle/axway-open-docs/page/docs/central/mesh_management/deploy-your-agents-with-the-amplify-cli/index.html)
```
Comment (lbadenhop) [inline content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md:64]:
```suggestion
* The alpha agent installation is documented here: [https://docs.axway.com/bundle/axway-open-docs/page/docs/central/mesh_management/deploy-your-agents-with-the-amplify-cli/index.html](/docs/central/mesh_management/deploy-your-agents-with-the-amplify-cli/index.html)
```

File: content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+* The alpha agent usage and configuration is documented here: [https://docs.axway.com/bundle/axway-open-docs/page/docs/central/mesh_management/discover-apis-and-services/index.html](https://docs.axway.com/bundle/axway-open-docs/page/docs/central/mesh_management/discover-apis-and-services/index.html)
```
Comment (lbadenhop) [inline content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md:65]:
```suggestion
* The alpha agent usage and configuration is documented here: [https://docs.axway.com/bundle/axway-open-docs/page/docs/central/mesh_management/discover-apis-and-services/index.html](/docs/central/mesh_management/discover-apis-and-services/index.html).
```

File: content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md [hunk 11]
```diff
@@ -0,0 +1,103 @@
...
+* The alpha Mesh Discovery agent limitations are noted below.
+
+## Fixed issues
+
+The following issues were fixed in this version of Amplify Central:
+
+* Previously, the Mesh Governance helm apic-hybrid chart installation step would not accept an alternate target namespace. Now, the `--namespace` option can use any any properly prepared namespace on the target cluster.
+* Previously, some Amplify Central CLI results from the amplify central get xxx commands did not correctly return their RESOURCE KIND and SCOPE KIND columns. Now, the RESOURCE KIND and SCOPE KIND columns are correctly populated for all Amplify Central resources.
+* Previously, the environment name was not reported for API transactions shown in the Amplify platform Visibility Dashboard. Now, API traffic reported by the v7 or AWS traceability agents as well as by the SaaS gateway will report the environment name for transactions visible in the API Observer and the platform Visibility Dashboard.
```
Comment (lbadenhop) [inline content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md:74]:
```suggestion
* Previously, the environment name was not reported for API transactions shown in the Amplify platform Visibility Dashboard. Now, API traffic reported by the API Gateway or AWS Traceability Agents as well as by the SaaS gateway will report the environment name for transactions visible in the API Observer and the platform Visibility Dashboard.
```

File: content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md [hunk 12]
```diff
@@ -0,0 +1,103 @@
...
+* The alpha agent usage and configuration is documented here: [https://docs.axway.com/bundle/axway-open-docs/page/docs/central/mesh_management/discover-apis-and-services/index.html](https://docs.axway.com/bundle/axway-open-docs/page/docs/central/mesh_management/discover-apis-and-services/index.html)
+* The alpha Mesh Discovery agent limitations are noted below.
+
+## Fixed issues
+
+The following issues were fixed in this version of Amplify Central:
+
+* Previously, the Mesh Governance helm apic-hybrid chart installation step would not accept an alternate target namespace. Now, the `--namespace` option can use any any properly prepared namespace on the target cluster.
+* Previously, some Amplify Central CLI results from the amplify central get xxx commands did not correctly return their RESOURCE KIND and SCOPE KIND columns. Now, the RESOURCE KIND and SCOPE KIND columns are correctly populated for all Amplify Central resources.
```
Comment (lbadenhop) [inline content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md:73]:
```suggestion
* Previously, some Amplify Central CLI results from the Amplify Central get xxx commands did not correctly return their RESOURCE KIND and SCOPE KIND columns. Now, the RESOURCE KIND and SCOPE KIND columns are correctly populated for all Amplify Central resources.
```

File: content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md [hunk 13]
```diff
@@ -0,0 +1,103 @@
...
+This version of Amplify Central has the following limitations:
+
+* API Observer:
+
+    * Users that are assigned the Consumer role cannot see their subscription usage on the API Observer screen.  
+
+* Axway Edge Gateway Agents:
+
+    * Discovery Agent only discovers APIs having PassThrough, API Key and Oauth security.
+    * Discovery Agent cannot expose discovered APIs in multiple teams, so the organization structure on API Manager is lost in Central. As a result, the API provider must create the team in Amplify Platform and share the API within appropriate teams.
```
Comment (lbadenhop) [inline content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md:87]:
```suggestion
    * Discovery Agent cannot expose discovered APIs in multiple teams, so the organization structure on API Manager is lost in Amplify Central. As a result, the API provider must create the team in Amplify Platform and share the API within appropriate teams.
```

File: content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md [hunk 14]
```diff
@@ -0,0 +1,103 @@
...
+    * Currently when an API is renamed on the v7 gateway the discovery agent is not able to recognize the API name change. This results in the API showing in Amplify Central with dual entries of both the originally discovered name and the newly changed name.
```
Comment (lbadenhop) [inline content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md:88]:
```suggestion
    * Currently when an API is renamed on the API Gateway the Discovery Agent is not able to recognize the API name change. This results in the API displaying in Amplify Central with dual entries of both the originally discovered name and the newly changed name.
```

File: content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md [hunk 15]
```diff
@@ -0,0 +1,103 @@
...
+    * Traceability Agent is not working in an externally managed topology deployment.
+
+* AWS Gateway agents:
+
+    * Discovery Agent is working with only one AWS Region (the one used when installing the agent).
+    * Discovery Agent does not associate the usage plan and API when a subscriber chooses a usage plan that is not already linked to the chosen API.
+
+* Azure agents:
+
+    * Discovery Agent is not managing revision and version yet
```
Comment (lbadenhop) [inline content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md:98]:
```suggestion
    * Discovery Agent is not managing revision and version yet.
```

File: content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md [hunk 16]
```diff
@@ -0,0 +1,103 @@
...
+    * Traceability agent is not reporting the App usage traffic
```
Comment (lbadenhop) [inline content/en/docs/central/amplify_relnotes/20210131_ampc_relnotes.md:99]:
```suggestion
    * Traceability agent is not reporting the App usage traffic.
```

**Review by lbadenhop — COMMENTED** [review]
@cbordier See review comments

**Comment by cbordier** [comment]
@lbadenhop, I am done with the review.
Can you merge it?

**Pull request merged**